pikaday.js

```diff
@@ -1238,6 +1332,7 @@
 
         show: function()
         {
+            document.body.appendChild(this.speakEl);
```

Comment:
We have to remove the element when the datepicker is closed. Currently, it will keep adding duplicate `speakEl` whenever reopen the datepicker